### PR TITLE
[ECH-452] feat: New Currency Support: 0ZK (Railgun)

### PR DESCRIPTION
**Pull request by udhriaznov**

### CODE REVIEW

**Review by iurevych — APPROVED** [review]
LGTM, but let's wait on approval from @unstoppabledomains/developer-tools-team

**Comment by iurevych** [comment]
Hey @sammyluo @DancingAxolotl this is an interesting token because I can see a Railgun wallet has public and private addresses. Which address should be support, if not both?
Railway: DeFi Privacy Wallet

File: resolver-keys.json [hunk 1]
```diff
@@ -2510,6 +2510,26 @@
...
+      "validationRegex": "(^0zk1)([qpzry9x8gf2tvdw0s3jn54khce6mua7l]{52})([qzyxg2vwsj5kc6u7])(unpd9kxwatw8)([qpzry9x8])([qpzry9x8gf2tvdw0s3jn54khce6mua7l]{57})",
+      "deprecated": false
+    },
+    "crypto.0ZK.version.MATIC.address": {
+      "deprecatedKeyName": "0ZK_MATIC",
+      "validationRegex": "(^0zk1)([qpzry9x8gf2tvdw0s3jn54khce6mua7l]{52})([qzyxg2vwsj5kc6u7])(unpd9kxwatw3)([qpzry9x8])([qpzry9x8gf2tvdw0s3jn54khce6mua7l]{57})",
+      "deprecated": false
+    },
+    "crypto.0ZK.version.0ZK.address": {
+      "deprecatedKeyName": "0ZK_MATIC",
```
Comment (sammyluo) [inline resolver-keys.json:2530]: Is it intended to have the `deprecatedKeyName` to be the same as L2525?

Comment (udhriaznov) [reply resolver-keys.json]: Oops, no, updated. Is it good now?

**Review by sammyluo — APPROVED** [review]

**Comment by iurevych** [comment]
@udhriaznov I'd ask the requestor

**Review by sammyluo — APPROVED** [review]
Jim replied in the Linear ticket and clarified that we only want to support private network.

PR LGTM

**Pull request merged**